services/movie.py

```diff
@@ -1,11 +1,13 @@
 from django.db.models import QuerySet
+from django.db import transaction
 
 from db.models import Movie
 
 
 def get_movies(
     genres_ids: list[int] = None,
     actors_ids: list[int] = None,
```

Comment:
Using mutable default arguments like lists is not recommended as it can lead to unexpected behavior. Consider using `None` as the default value and initializing the list within the function.